lib/eldap/src/eldap.erl

```diff
@@ -36,6 +36,22 @@ Type definitions that are used more than once in this module:
 - **`referrals() =`** - `[Address = string()]` The contents of `Address` is
   server dependent.
 """.
+
+-type search_option() :: {base, string()} | {filter, filter()} |
+                         {scope, scope()} | {attributes, [string()]} |
+                         {deref, dereference()} | {types_only, boolean()} |
+			 {timeout, integer()}.
+-type string_part() :: initial | any | final.
+-type return_value() :: ok | {error, term()}.
```

Comment:
In our meeting we discussed this and that it should not really be used.
I suggest remove it and use the actual return values in the spec's.
Also note that the "type" was previously different: ok | {ok, Value} | {error, Reason}

Reply:
Lets keep the convention that we use " ok | {ok, Value} | {error, Reason}", instead of introduce a new "return_values()" type .